plugin/src/test/groovy/com/github/triplet/gradle/play/PlayPublishTaskTest.groovy

```diff
@@ -111,6 +111,23 @@ class PlayPublishTaskTest {
         doReturn(editCommitMock).when(editsMock).commit(anyString(), anyString())
     }
 
+    @Test
+    void whenPublishing_withOverrideFolder_UploadsOnlySpecifiedFiles() {
+        def project = TestHelper.evaluatableProject()
+        project.play {
+            buildInputFolder new File(TestHelper.FIXTURE_WORKING_DIR, "override_output")
+        }
+        project.evaluate()
+
+        setMockPublisher(project.tasks.publishApkRelease)
+        publishApk(project.tasks.publishApkRelease, "example_1.apk")
```

Comment:
This test looks a little suspicious. Shouldn't that filename be automatically retrieved when calling publish? Seems like we're testing the implementation of `AndroidPublisher.Edits.publishApk(editId: String, apkFile: FileContent)` which is too late. The `inputApks` property is read from the main `@TaskAction`

Reply:
I was able to add a test for the empty/non-existent directory that is set, but I still can't figure out how to trigger the `upload` action for this test.